README.md

```diff
@@ -15,8 +15,7 @@ Omes (pronounced oh-mess) is the Hebrew word for "load" (עומס).
     available for download that you can use with `git apply`
 - [Java](https://openjdk.org/) 8+
 - TypeScript: [Node](https://nodejs.org) 16+
-- [Python](https://www.python.org/) 3.10+
-  - [Poetry](https://python-poetry.org/): `poetry install`
+- [uv](https://docs.astral.sh/uv/)
```

Comment:
Do they no longer need Python installed? So `uv` installs it lazy? If so, can make it clear `uv` is only for the purposes of Python like TypeScript does with Node:

```suggestion
- Python: [uv](https://docs.astral.sh/uv/)
```

Reply:
Right, `uv` installs pythons and manages which is active.